release_notes/v2.4.1.md

```diff
@@ -0,0 +1,144 @@
+v2.4.1 Release Notes - December 17, 2021
+========================================
+
+
+Improvements
+------------
+
+**peer - Chaincode as a service builder delivered with Fabric release artifacts and in fabric-peer docker image**
+
+Starting in v2.0 chaincode can be run as a service by utilizing the external builder pattern for chaincodes.
+Running 'chaincode as a service' has advantages in Kubernetes and other deployment environments since the chaincode can be managed independently rather than requiring the peer to build chaincode images and start chaincode containers at runtime.
+The external builder pattern required you to deliver a builder script or program alongside the peer.
+Starting in v2.4.1 an external builder for 'chaincode as a service' is available in the Fabric release artifacts and is pre-configured with the fabric-peer docker image,
+removing the need to build your own external builder and repackage and configure the peer.
+
+**peer - Gateway service now randomizes endorsement layouts**
+
+For improved load balancing of endorsement requests and fairer workload distribution across organizations,
+the gateway service now randomizes the peers it connects to for every transaction.
+
+**peer - Gateway service concurrency limits**
+
+Add support for setting gateway service concurrency limits, similar to how concurrency limits are
+set for the existing endorser and deliver services.
+Concurrency limits ensure that a peer does not process more than the configured number of requests,
+reducing the chance of a peer getting overloaded with more requests than it can process.
+The peer's core.yaml 'peer.limits.concurrency.gatewayService' setting defaults to 500.
+
+Fixes
+-----
+
+**peer - Fix for orderer endpoint updates in channel configuration transactions**
+
+When processing a channel configuration transaction, the gateway service was setting orderer endpoints based on the
```

Comment:
Gateway service is new, so is there really a "was setting" behavior - in a prior version of the service - that needs publishing?

Reply:
so maybe this is just "the default behavior" or the "new programming model" rather than "the gateway service" doing this.

Reply:
These are the v2.4.1 release notes, so the references to prior version are references to v2.4.0 release.